modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -153,6 +158,7 @@ export default {
       },
       triggerValidation: false,
       original: null,
+      originalDoc: {},
```

Comment:
ℹ️ Use an object in order to keep the ref "alive".
Providing `original` directly would result in having a `null` injected value.